includes/class-wcmp-data.php

```diff
@@ -31,7 +31,6 @@ class WCMP_Data
     public const HAS_MULTI_COLLO = false;
 
     public const DEFAULT_COUNTRY_CODE = "BE";
-    public const DEFAULT_CARRIER      = BpostConsignment::CARRIER_NAME;
```

Comment:
Waarom is dit weg? Lijkt me handig als dit de default default carrier is en een fallback voor wanneer die setting niet is ingevuld :P